codewof/programming/content/en/happy-birthday/solution.py

```diff
@@ -0,0 +1,4 @@
+birth_year = int(input("What year were you born? "))
+year = int(input("What year is it now? "))
+age = year - birth_year
+print("Your {}th birthday is this year.".format(age))
```

Comment:
Is print formatting something teachers are likely to be familiar with and need to know? My first thought was that this seemed difficult, but it's been a long time since I looked at any of the high school material (or print formatting actually!) so I might just be out of date

Reply:
After discussing string formatting at our meeting this morning, I'm going to alter this question slightly to make the string formatting simpler.

Reply:
Should be okay, though it is at the harder end of the scale. They can answer using string concatenation if they don't know about string formatting, it just means they will need to convert the years from strings to ints and then the age from int to string.

Reply:
Sorry I read this right after pushing the change! The version I pushed has them do `print("On your birthday this year you will be turning", age)`

Would you like me to change it back?

Reply:
I actually like your version better because it doesn't assume that the suffix is `th` :)